### PR TITLE
Changelog for fix to #4424

### PR DESCRIPTION
**Pull request by andybalaam**

### CODE REVIEW

File: crates/matrix-sdk-crypto/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased] - ReleaseDate
 
+- Fix [#4424](https://github.com/matrix-org/matrix-rust-sdk/issues/4424) Failed storage upgrade for
+  "PreviouslyVerifiedButNoLonger"
```
Comment (poljar) [inline crates/matrix-sdk-crypto/CHANGELOG.md:10]: That's pretty terse, can we expand on what and how this affects users. Also all the other entries link a PR or commit at the end, could we stick to that convention.

**Comment by richvdh** [comment]
This should maybe target release-for-crypto-wasm-11, and be forward-merged from there?

File: crates/matrix-sdk-crypto/CHANGELOG.md (hunk 1 above)
Comment (andybalaam) [reply crates/matrix-sdk-crypto/CHANGELOG.md]: Updated the description, rebased against `release-for-crypto-wasm-11` and retargeted this PR. Will open a new PR for the merge into main.

Comment (andybalaam) [reply crates/matrix-sdk-crypto/CHANGELOG.md]: New PR into main is https://github.com/matrix-org/matrix-rust-sdk/pull/4434

**Review by poljar — APPROVED** [review]

**Pull request merged**